src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -99,6 +100,7 @@ class SaaSRequest(BaseModel):
     pagination: Optional[Strategy]
     grouped_inputs: Optional[List[str]] = []
     ignore_errors: Optional[bool] = False
+    rate_limits: Optional[List[RateLimitConfig]]
```

Comment:
not sure on the best place in the code to put this comment -- but ideally we'd have at least one "end-to-end" test that affirms the `rate_limit` config here is getting properly picked up and injected into the saas execution framework.

i'm imagining that we'd add one or more rate limits to the `saas_example_config.yml`, and we'd _probably_ need to do some mocking to make sure it's getting taking an effect the way we want it to. even without the mocking and just making sure it's picked up at all on the saas config i think would be useful.

this would be helpful to ensure no regressions are accidentally introduced going forward, and also for providing a user-facing example on how rate limits can be actually configured on saas connectors.